src/internal/operations/backup_integration_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"time"
 
 	"github.com/google/uuid"
+	msdrive "github.com/microsoftgraph/msgraph-sdk-go/drive"
```

Comment:
nit: would like to reduce unnecessary aliasing.
```suggestion
	"github.com/microsoftgraph/msgraph-sdk-go/drive"
```